docs/website/02_examples/vonMisesTruss.md

```diff
@@ -59,12 +59,12 @@ This is implemented as depicted in the following:
 const int loadSteps = 10;
 Eigen::Matrix3Xd lambdaAndDisp;
 lambdaAndDisp.setZero(Eigen::NoChange, loadSteps + 1);
-auto lvkObserver = std::make_shared<Ikarus::GenericObserver<Ikarus::ControlMessages>>(
-    Ikarus::ControlMessages::SOLUTION_CHANGED, [&](int step) {
-      lambdaAndDisp(0, step) = lambda;
-      lambdaAndDisp(1, step) = d[2];
-      lambdaAndDisp(2, step) = d[3];
-    });
+auto lvkObserver = std::make_shared<GenericObserver<ControlMessages, ControlState>>(
```

Comment:
The observer should be able to get as a template just the observable object.

Reply:
But I remember that we had reasons to keep the templates of the observer like this, I dont remember. Can you?

Reply:
Ah yes you are right. I fixed this here `ikarus/utils/observer/genericobserver.hh` where we have 
```cpp
template <typename MT, typename ST>
class GenericObserver : public IObserver<IObservable<MT, ST>>
```
So accordingly in the example, observable must be passed. I will fix it